lib/galaxy/tools/recommendations.py

```diff
@@ -27,86 +27,124 @@ def __init__(self):
         self.reverse_dictionary = dict()
         self.all_tools = dict()
         self.tool_weights_sorted = dict()
-        self.session = None
-        self.graph = None
         self.loaded_model = None
         self.compatible_tools = None
         self.standard_connections = None
         self.max_seq_len = 25
+        self.ff_dim = 128
+        self.embed_dim = 128
+        self.num_heads = 4
+        self.dropout = 0.1
```

Comment:
These could probably be class attributes?

Reply:
yes, good idea, fixed. Thanks